File: C/FizzBuzz.c
#include <stdio.h>

int main(char argc[], char **argv) {
    void FizzBuzz() {
        char t = 0;
        for (unsigned char i = 1; i <= 100; ++i, t = 2) {
            (i % 3) ? --t : printf("Fizz");
            (i % 5) ? --t : printf("Buzz");
            if (!t) printf("%d", i);
            printf("\n");   
        }
    }
    FizzBuzz();
}

File: Elixir/FizzBuzz.ex
defmodule FizzBuzz do
  def play(min, max) do
  Enum.each(min..max, fn(num)-> play(num) end )
  end
  def play(num) when rem(num, 15) == 0 do
    IO.puts "FizzBuzz"
  end
  def play(num) when rem(num, 3) == 0 do
    IO.puts "Fizz"
  end
  def play(num) when rem(num, 5) == 0 do
    IO.puts "Buzz"
  end
  def play(num) when rem(num, 15) == 0 do
    IO.puts "FizzBuzz"
  end
  def play(num) do
    IO.puts num
  end
end
FizzBuzz.play 1,100
